src/App.vue

```diff
@@ -31,7 +34,15 @@ export default {
   '$route.meta'({hideNavigation}){
     this.showNav = !hideNavigation;
   }
- }
+  },
+  created() {
+    //Pass store to window object on testing
+    if (window.cypress !== null) { 
```

Comment:
I think it's a capital C and that it will be undefined rather than null when missing
```suggestion
    if (window.Cypress) { 
```